includes/services/core/http-basic.php

```diff
@@ -206,7 +206,7 @@ function request( $url, array $params = array() ) {
 
 		Keyring_Util::debug( $res );
 		$this->set_request_response_code( wp_remote_retrieve_response_code( $res ) );
-		if ( wp_startswith( wp_remote_retrieve_response_code( $res ), '2' ) ) {
```

Comment:
we didn't think wp_startswith() should be in the plugin as it is a Jetpack/WP.com function